src/System.Private.CoreLib/src/Internal/Console.cs

```diff
@@ -27,7 +28,7 @@ public static unsafe void Write(string s)
             }
         }
 
-        public static void WriteLine(string s) =>
+        public static void WriteLine(string? s) =>
```

Comment:
It's a little strange that `WriteLine(string)` accepts `null` but `Write(string)` does not.  That said, even though this type is "public", it isn't really, so it's not a big deal.